Modules/SubModule_Import/xASL_imp_NII2BIDS_Session.m

```diff
@@ -30,6 +30,10 @@
     %% Unpack bidsLabel struct
     subjectLabel = bidsLabel.subject;
     visitLabel = bidsLabel.visit;
+    
+    %% Print current session which is being converted
```

Comment:
`%% Print current session which is being converted`

should be

`%% Print the session that is being converted`

(learn English grammar ;)